app/templates/components/expectation-modal.hbs

```diff
@@ -4,12 +4,12 @@
     <p>{{t (compute (action textTag  "description"))}}</p>
   </div>
 
-  {{#if (compute (action isControl))}}
-    <ControlGroupExpectations
+  {{#if (compute (action shouldShowNonScoredExpects))}}
+    <NonScoredExpectations
```

Comment:
BIEN!!

Reply:
En retropctiva, la doble negación quizás está mal, lo que se debería preguntar es por el positivo, así también los tests quedan más copados: "if should show scored expects, muestro las scored"